php-checks/src/main/resources/org/sonar/l10n/php/rules/php/S1313.html

```diff
@@ -0,0 +1,27 @@
+<p>Hardcoding an IP address into source code is a bad idea for several reasons:</p>
+<ul>
+  <li> a recompile is required if the address changes </li>
+  <li> it forces the same address to be used in every environment (dev, sys, qa, prod) </li>
+  <li> it places the responsibility of setting the value to use in production on the shoulders of the developer </li>
+  <li> it allows attackers to decompile the code and thereby discover a potentially sensitive address </li>
```

Comment:
Reasons 1 and 4 are about compiling and decompiling which don't make sense for PHP AFAIK.